addons/account_edi_facturx/models/account_edi_format.py

```diff
@@ -127,7 +123,15 @@ def _is_facturx(self, filename, tree):
     def _create_invoice_from_xml_tree(self, filename, tree):
         self.ensure_one()
         if self._is_facturx(filename, tree):
-            return self._import_facturx(tree, self.env['account.move'])
+            invoice = self.env['account.move'].create({})
+            # During the process, if an Error is raised, the invoice isn't rollback and is persisted in the DB
+            # Thus we clean the invoice before propagating the error
+            try:
+                res = self._import_facturx(tree, invoice)
+            except (ValidationError, UserError) as e:
```

Comment:
Is that only valid for those exceptions?
Since you don't use `e`, no need to create the variable, I think it will be blocked by the CI during fw-port to master.

Reply:
I just want to catch Error that would be due to constraints.

I don't need the e because I raise again, I will get rid of it.